lib/msf/util/exe.rb

```diff
@@ -1744,7 +1967,7 @@ def self.win32_rwx_exec_thread(code, block_offset, which_offset='start')
 ;     pop eax                ; Skip
       pop eax                ; Skip
       popad                  ; Get our registers back
-;     sub esp, 44             ; Move stack pointer back past the handler
+;     sub esp, 44            ; Move stack pointer back past the handler
```

Comment:
I dislike this change because it makes it harder to see that the instruction is commented out. It might be better to just axe these commented-out code lines altogether.